applications/osb-portal/src/components/drawer/WorkspaceDrawer.tsx

```diff
@@ -81,41 +166,101 @@ export const WorkspaceDrawer = (props: any) => {
 
   const handleToggleDrawer = () => props.onToggleDrawer();
 
+  const drawerContent = (open)
+  ? <>
+      <TreeItem
+        nodeId="1"
+        label={
+          <div>Workspace XYZ
+            <SvgIcon className={classes.svgIconRight} viewBox={'0 0 16 16'}>
+              <path d="M9.5 8.37891C9.97396 8.37891 10.375 8.55208 10.7031 8.89844C11.0312 9.22656 11.1953 9.61849 11.1953 10.0742C11.1953 10.5482 11.0221 10.9583 10.6758 11.3047C10.3477 11.6328 9.95573 11.7969 9.5 11.7969C9.04427 11.7969 8.64323 11.6328 8.29687 11.3047C7.96875 10.9583 7.80469 10.5482 7.80469 10.0742C7.80469 9.89193 7.8138 9.76432 7.83203 9.69141L3.70312 7.28516C3.35677 7.59505 2.95573 7.75 2.5 7.75C2.02604 7.75 1.61589 7.57682 1.26953 7.23047C0.923177 6.88411 0.75 6.47396 0.75 6C0.75 5.52604 0.923177 5.11589 1.26953 4.76953C1.61589 4.42318 2.02604 4.25 2.5 4.25C2.95573 4.25 3.35677 4.40495 3.70312 4.71484L7.80469 2.33594C7.76823 2.15365 7.75 2.01693 7.75 1.92578C7.75 1.45182 7.92318 1.04167 8.26953 0.695312C8.61589 0.348958 9.02604 0.175781 9.5 0.175781C9.97396 0.175781 10.3841 0.348958 10.7305 0.695312C11.0768 1.04167 11.25 1.45182 11.25 1.92578C11.25 2.39974 11.0768 2.8099 10.7305 3.15625C10.3841 3.5026 9.97396 3.67578 9.5 3.67578C9.0625 3.67578 8.66146 3.51172 8.29687 3.18359L4.19531 5.58984C4.23177 5.77214 4.25 5.90885 4.25 6C4.25 6.09115 4.23177 6.22786 4.19531 6.41016L8.35156 8.81641C8.67969 8.52474 9.0625 8.37891 9.5 8.37891Z" fill="white"/>
+            </SvgIcon>
+          </div>
+        }
+      >
+        <Divider />
+          <ListItem button={true}>
+              <ListItemIcon><AddIcon /></ListItemIcon>
+              <ListItemText primary={'Add resource'} />
+          </ListItem>
+        <Divider />
+        <TreeItem nodeId="3" label={<LinkItem name="Ferguson 0.nwb" />} />
```

Comment:
The tree was out of scope on this round. A dedicated component will show workspace content. Good to have it here to see how it looks anyway for now.